rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

```diff
@@ -97,6 +98,27 @@ Learn how to xref:processautomation-prepare-project-application.adoc[manage appl
 . In the form *<bot name>* click on all service times and applications you want to assign.
 . Click *Save*.
 
+== Update a Bot
+
+Update your bots to the latest version to get the newest features, bugfixes, and security enhancements. The bot completes all processes before updating, and resumes them afterwards to avoid data loss.
```

Comment:
Make sure this terminology (completes processes) is the same in the UI text.

Reply:
replaced with finishes to match UI text